packages/components/src/components/tooltip/tooltip.css

```diff
@@ -0,0 +1,163 @@
+host*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+:host {
+  /*tooltip*/
+  --border-radius: var(--scl-radius-4);
+  --background-color: var(--scl-color-grey-60);
+  --color: var(--scl-color-white);
+  --font-weight: var(--scl-font-weight-regular);
+  --font-size: var(--scl-font-size-16);
+  --line-height: 1.4;
+  --padding: var(--scl-spacing-2) var(--scl-spacing-8);
```

Comment:
can we use `--spacing`?

Reply:
Wouldn't it be better to keep padding, due to consistency ? Because we use for all other variables names exactly the name for the matching property

Reply:
Yes, but we also use `--spacing` in almost all the other components. Maybe it wasn't the best decision using `spacing` in the first place, but it's only because of consistency within the library that I'm suggesting to use `--spacing`.

The reason for using `spacing` is to keep it independent from the internal implementation, so to avoid having something like `margin: var(--padding)` if you ever need to refactor the component and use margins instead of padding, or viceversa.

All other suggestions follow this same principle, consistency across components. Thanks for asking 🙏